scipy/stats/_distn_infrastructure.py

```diff
@@ -1660,6 +1648,42 @@ def __init__(self, momtype=1, a=None, b=None, xtol=1e-14,
                 dct = dict(distcont)
                 self._construct_doc(docdict, dct.get(self.name))
 
+    def __getstate__(self):
+        dct = self.__dict__.copy()
+
+        # these methods will be remade in __setstate__
+        # _random_state attribute is taken care of by rv_generic
+        attrs = ["_parse_args", "_parse_args_stats", "_parse_args_rvs",
+                 "_cdfvec", "_ppfvec", "vecentropy", "generic_moment"]
+        [dct.pop(attr, None) for attr in attrs]
+        return dct
+
+    def __setstate__(self, state):
+        dct = state
+        self.__dict__.update(dct)
```

Comment:
is `dct` a leftover? cf similar stanzas in `rv_discete`

Reply:
changed